### ISSUE TITLE
Add documentation on loading bitmaps on MAUI

### ISSUE THREAD

**Issue by juliusdeblaaij** [Mapsui/Mapsui#2743]
**The problem**
When trying to load a bitmap image on MAUI for a symbol style, I keep getting errors that my image cannot be found.
I follow the Beta documentation as i found here:
https://github.com/Mapsui/Mapsui/blob/37f6020629a75e2528a7332582c91a5368259ad8/docs/general/markdown/imagesource.md?plain=1#L25

I tried various versions of this method:
myStyle.ImageSource = "embedded://Mapsui.Resources.Images.Pin.svg"
but neither images in the Resources/Images or Resources/Raw folder can be found.

**The proposed solution**
Please document a example of loading a local image from Resources/Images or Resources/Raw on MAUI.

**Alternative solutions**
Explain how an embedded:// URL can be used to load a Resources/Image or Resources/Raw.

**Additional context**
I tried various versions, with or without the assembly name, with our withour Resources etc.

**Comment by juliusdeblaaij** [comment]
Here is a bit of trace from debugging the application when it tries to use an ambedded image (png) as an image source:

```
 Exception: System.Exception: Could not find the embedded resource in the current assemblies. ImageSource: 'embedded://mauireactortest.resources.images.warp_marker.png/'. Other embedded resources in matching assemblies: 
```

What I find noteworthy as that there are no other embedded resources found either 9even though I have multiple images under Resources/Images)

**Comment by inforithmics** [comment]
You can view the pathes of the embedded images with a tool like dotpeek.

https://www.jetbrains.com/de-de/decompiler/

**Comment by juliusdeblaaij** [comment]
Thank you!
After adding a line like this for each image inside my Csproj:
`<EmbeddedResource Include="Resources\Images\left_area_marker.png" />` inside an `<ItemGroup>`, the image shows correctly.
Is there interest for me to add a description of this to the docs?
I was thinking on this page:
https://github.com/Mapsui/Mapsui/blob/main/docs/general/markdown/imagesource.md
To help out other beginners :)

Kind regards,

Julius

**Comment by inforithmics** [comment]
Yes would be nice If you could do a pull request or write something for improving the documentation.
